src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -1390,60 +1416,20 @@ namespace winrt::Microsoft::Terminal::Control::implementation
     // Arguments:
     // - noteNumber - The MIDI note number to be played (0 - 127).
     // - velocity - The force with which the note should be played (0 - 127).
-    // - duration - How long the note should be sustained (in microseconds).
-    void ControlCore::_terminalPlayMidiNote(const int noteNumber, const int velocity, const std::chrono::microseconds duration)
+    // - duration - How long the note should be sustained.
+    void ControlCore::_terminalPlayMidiNote(const int noteNumber, const int velocity, const MidiDuration duration)
     {
-        // We create the audio instance on demand, and lock it for the duration
-        // of the note output so it can't be destroyed while in use.
-        auto& midiAudio = _getMidiAudio();
-        midiAudio.Lock();
-
         // We then unlock the terminal, so the UI doesn't hang while we're busy.
         auto& terminalLock = _terminal->GetReadWriteLock();
         terminalLock.unlock();

```

Comment:
With the code above removed, this comment doesn't make sense. Maybe just need to drop the "then".